src/pages/edit-profile/EditProfile.styles.ts

```diff
@@ -1,13 +1,18 @@
+import palette from '~/styles/app-theme/app.pallete'
 import { TypographyVariantEnum } from '~/types'
 
 export const styles = {
+  sectionsWrapper: {
+    pr: { lg: '156px' },
+    pl: { lg: '156px' }
+  },
```

Comment:
Maybe it would be better to use `px: { lg: '156px' }`?

Reply:
Thanks! I've fixed it